docs/standard/serialization/system-text-json/snippets/how-to-contd/csharp/CustomConverterUnixEpochDateNoZone.cs

```diff
@@ -1,30 +1,50 @@
 ﻿using System.Globalization;
 using System.Text.Json;
-using System.Text.Json.Serialization;
 using System.Text.RegularExpressions;
+using Newtonsoft.Json;
 
 namespace CustomConverterUnixEpochDateNoZone
 {
     class Program
     {
-        public static void Main()
+        public static void Main(DateTime date)
```

Comment:
It seems somewhat odd to me that we're repurposing a `Main` function in a `Program` class to be anything other than the entrypoint of a console app. I would recommend keeping the `Main()` function here and factor the individual workloads into parametric functions with different naming.